docs/core/introduction.md

```diff
@@ -179,231 +76,172 @@ A .NET app is built from source code by using [MSBuild](/visualstudio/msbuild/ms
 
-### Working with unmanaged resources
+The runtime exposes various [diagnostics](https://docs.microsoft.com/dotnet/core/diagnostics/) services and APIs for debuggers, [dump](diagnostics/dumps/) and [tracing](diagnostics/logging-tracing/) tools, and [observability](diagnostics/#instrumentation-for-observability). The observability implementation is primarily [built around OpenTelemetry](https://devblogs.microsoft.com/dotnet/opentelemetry-net-reaches-v1-0/), enabling [flexible application monitoring](https://devblogs.microsoft.com/dotnet/announcing-dotnet-monitor-in-net-6/) and site reliability engineering (SRE).
 
-Sometimes code needs to reference *unmanaged resources*. Unmanaged resources are resources that aren't automatically maintained by the .NET runtime. For example, a file handle is an unmanaged resource. A <xref:System.IO.FileStream> object is a managed object, but it references a file handle, which is unmanaged. When you're done using the <xref:System.IO.FileStream>, you need to explicitly release the file handle.
+The runtime offers low-level C-style interop functionality, via a combination of [P/Invoke](../standard/native-interop/index.md), value types, and the ability to [blit](../framework/interop/blittable-and-non-blittable-types) values across the native/managed-code boundary.
 
-In .NET, objects that reference unmanaged resources implement the <xref:System.IDisposable> interface. When you're done using the object, you call the object's <xref:System.IDisposable.Dispose> method, which is responsible for releasing any unmanaged resources. The .NET languages provide a convenient `using` statement ([C#](../csharp/language-reference/keywords/using.md), [F#](../fsharp/language-reference/resource-management-the-use-keyword.md), [VB](../visual-basic/language-reference/statements/using-statement.md)) that ensures the `Dispose` method is called.
+## Languages
 
-For more information, see [Cleaning up unmanaged resources](../standard/garbage-collection/unmanaged.md).
+The CLR is designed to support multiple programming languages. C#, F#, and Visual Basic languages are supported by Microsoft and are designed in collaboration with the community.
 
-## Deployment models
+* [C#](../csharp/index.yml) is a modern, object-oriented, and type-safe programming language. It has its roots in the C family of languages and will be immediately familiar to C, C++, Java, and JavaScript programmers.
 
-.NET apps can be published in two different modes:
+* [F#](../fsharp/index.yml) is an interoperable programming language for writing succinct, robust, and performant code. F# programming is data-oriented, where code involves transforming data with functions.
+  
+* [Visual Basic](../visual-basic/index.yml) uses a more verbose syntax that is closer to ordinary human language. It can be an easier language to learn for people new to programming.
```

Comment:
Is that true, or just a hypothesis? I have not seen evidence of this (although, I could well simply have not seen it)

The principal reason to use VB I would think is existing familiarity with VB or other Basic-style languages.

Reply:
This was the text from before. Thoughts @KathleenDollard?